normandy/recipes/api/v1/views.py

```diff
@@ -168,6 +169,7 @@ class ClassifyClient(views.APIView):
     permission_classes = []
     serializer_class = ClientSerializer
 
+    @cache_control(no_cache=True, no_store=True, must_revalidate=True)
```

Comment:
Oops, my comment didn't go through with the review.

THis looks fine, but I think you could just use the [never_cache](https://docs.djangoproject.com/en/2.1/topics/http/decorators/#caching) decorator.